### DISCUSSION TITLE
How to print footer of each revealjs slides?

### DISCUSSION THREAD

**Discussion by drganghe** [quarto-dev/quarto-cli#3319] [Q&A]
I'm trying to print my course slides to pdf, below is an example:

https://drganghe.github.io/est603-energy-systems-analysis-2022-fall/lectures/lecture12/#/title-slide

I have the footer setting in the format.

format:
    revealjs:
         footer: <https://drganghe.github.io/est603-energy-systems-analysis-2022-fall/>

Then the footer overrides the footers of individual slides when printing pdf using e or tools.

Is there any way to fix this? Thanks.

**Comment by mcanouil** [comment]
Hum, I think the issue is not really about an overwrite, because the footer does not seem to be tied up to slides (using `?print-pdf`).
Now, the question is to know if this is a bug n Reveal.js or Quarto.

Meanwhile, are you aware of the `attribution` Quarto extension? <https://github.com/quarto-ext/attribution>.

**Reply by drganghe** [reply]
Thank you for sharing, the `attribution` extension is good. 

I guess I can still do `footer` slide by slide, but I hope this bug can be fixed.

**Reply by mcanouil** [reply]
There is definitely something wrong with the footer in Reveal.js slides, I just had a similar issue.
I'll try to see if can more clearly identify the issue.
